Apps/Src/Task_FaultState.c

```diff
@@ -0,0 +1,74 @@
+#include "os.h"
+#include "Tasks.h"
+#include "BSP_Contactor.h"
+#include "BSP_Lights.h"
+#include "Voltage.h"
+#include "BSP_Fans.h"
+#include "Temperature.h"
+#include "Amps.h"
+#include "EEPROM.h"
+#include "BSP_WDTimer.h"
+#include "CLI.h"
+#include "CANbus.h"
+#include "BSP_UART.h"
+#include "config.h"
+//Manthan Upadhyaya wuz here: 10/2020
```

Comment:
I originally did this because I'd have issues reading code and I didn't know who to ask. With the documentation we are doing this is unnecessary so I can remove it.

Reply:
Fixed in #274